### PR TITLE
Benchmarks for seq_to_hashes in protein mode

### LINKED ISSUE #1937: benchmark cmd `tox -e asv` requires `python=3.10`
When benchmarking using `tox -e asv` on my `python 3.9` env, it requires a specific python version of `3.10` as described in the following error.

```sh
· No executable found for python 3.10
· No environments selected
```

### PR DESCRIPTION
**Pull request by luizirber**
Add some rust benchmarks for seq_to_hashes in protein mode, so we don't regress after the awesome progress in #1938 

Other fixes:
- Default python in `asv` to the one where `asv` is running (fixes https://github.com/sourmash-bio/sourmash/issues/1937)
- Make `ComputeParameters` initliazer use explicit keywords (and fix `__repr__`)

### CODE REVIEW

File: src/sourmash/command_compute.py [hunk 1]
```diff
@@ -326,7 +326,16 @@ def save_sigs_to_location(siglist, save_sig):
 class ComputeParameters(RustObject):
     __dealloc_func__ = lib.computeparams_free
 
-    def __init__(self, ksizes, seed, protein, dayhoff, hp, dna, num_hashes, track_abundance, scaled):
+    def __init__(self,
+                 ksizes=(21, 31, 51),
```
Comment (ctb) [inline src/sourmash/command_compute.py:336]: do we want to put a `*` in here? it would break some more code but would be good.

Happy to submit my own PR to do this after this is merged, if you like.

**Review by ctb — APPROVED** [review]
LGTM!!

File: src/sourmash/command_compute.py (hunk 1 above)
Comment (luizirber) [reply src/sourmash/command_compute.py:336]: Good point. I wanted the `__repr__` to make it easier to figure out what values were being passed in the protein case, and explicit values here to make sure they matched the defaults in Rust.

(it is funny that you also implemented `__repr__` recently =P)

Comment (luizirber) [reply src/sourmash/command_compute.py:336]: I put the `*` in the latest commit, but technically that's a breaking change in the API :see_no_evil:

Comment (luizirber) [reply src/sourmash/command_compute.py:336]: (but agree it is a good thing to put the `*`)

Comment (ctb) [reply src/sourmash/command_compute.py:336]: I have generally refrained from putting any guarantees on APIs that aren't exposed at the top level sourmash namespace, FWIW. Well, and the CLI.
